src/pages/Food/BurgerMenu/index.jsx

```diff
@@ -0,0 +1,52 @@
+import { stack as Menu } from 'react-burger-menu';
+import { Link } from 'react-router-dom';
+import SMenu from './style';
+
+function BurgerMenu() {
+  return (
+    <SMenu>
+      <Menu>
+        <ul>
+          <li>
+            <Link
+              to="/"
+              className="home"
+              style={{ color: 'inherit', textDecoration: 'inherit' }}
```

Comment:
Hum, style inside html tag ??
The GlobalStyle have already
a {
    text-decoration: none;
    color:black;
  }
  
  ? to check